tests/contrib/test_appengine.py

```diff
@@ -557,6 +656,24 @@ def tearDown(self):
         self.testbed.deactivate()
         httplib2.Http = self.httplib2_orig
 
+    def test_in_error(self):
+        # Note: as far as I can tell this branch is never reached. _in_error
```

Comment:
Fixed. As far as filing a bug, this entire module probably needs to be re-thought. This is minor compared to all of the other nonsense going on in here.